.travis.yml

```diff
@@ -1,38 +1,75 @@
+sudo: required
+
 matrix:
   include:
-    # works on Precise and Trusty
     - os: linux
       addons:
         apt:
           sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-5
-      env:
-         - MATRIX_EVAL="CC=gcc-5 && CXX=g++-5"
-
-#os: linux
-dist: trusty
-language: cpp
-compiler: gcc-5
-group: deprecated-2017Q2
+            - sourceline: 'deb http://archive.neon.kde.org/user/lts/ xenial main binary-amd64'
+
+      dist: xenial
+      language: cpp
+      compiler: gcc-5
+      group: deprecated-2017Q2
+    - os: osx
```

Comment:
Let us supply the default language for OS X builds to be C++ as well. By default, Travis assumes the language to be Ruby and tries to be helpful by setting up Ruby-specific environment, which we don't really need:
```bash
$ rvm use default
Using /Users/travis/.rvm/gems/ruby-2.4.2
ruby.versions
$ ruby --version
ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin16]
$ rvm --version
rvm 1.29.3 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
$ bundle --version
Bundler version 1.16.0
$ gem --version
2.6.13
```

Reply:
Nevermind, I am adding this change in my `ElMaven-CD` branch.